etna/search/tests/test_views.py

```diff
@@ -606,348 +595,587 @@ def test_current_bucket(self):
+                "customDimension6": "",
+                "customDimension7": "",
+                "customDimension8": "",
+                "customDimension9": "",
+                "customDimension10": "",
+                "customDimension11": "",
+                "customDimension12": "",
+                "customDimension13": "",
+                "customDimension14": "",
+                "customDimension15": "",
+                "customDimension16": "",
+                "customDimension17": "",
+                "customMetric1": 0,
+                "customMetric2": 0,
+            },
+        )
 
     @responses.activate
     def test_datalayer_featured_search(self):
-        path = f"{settings.BASE_DIR}/etna/search/tests/fixtures/featured_search.json"
-        with open(path, "r") as f:
-            responses.add(
-                responses.GET,
-                "https://kong.test/data/searchAll",
-                json=json_module.loads(f.read()),
-            )
-
-        response = self.client.get("/search/featured/")
-
-        self.assertTemplateUsed(response, "search/featured_search.html")
-
-        html_decoded_response = response.content.decode("utf8")
-        desired_datalayer_script_tag = """<script id="gtmDatalayer" type="application/json">{"contentGroup1": "Search", "customDimension1": "offsite", "customDimension2": "", "customDimension3": "FeaturedSearchView", "customDimension4": "", "customDimension5": "", "customDimension6": "", "customDimension7": "", "customDimension8": "All results: none", "customDimension9": "*", "customDimension10": "", "customDimension11": "", "customDimension12": "", "customDimension13": "", "customDimension14": "", "customDimension15": "", "customDimension16": "", "customDimension17": "", "customMetric1": 10001, "customMetric2": 0}</script>"""
-        self.assertIn(desired_datalayer_script_tag, html_decoded_response)
+        self.assertDataLayerEquals(
+            path=reverse("search-featured"),
+            query_data={},
+            api_resonse_path=f"{settings.BASE_DIR}/etna/search/tests/fixtures/featured_search.json",
+            expected={
+                "contentGroup1": "Search",
+                "customDimension1": "offsite",
+                "customDimension2": "",
+                "customDimension3": "FeaturedSearchView",
+                "customDimension4": "",
+                "customDimension5": "",
+                "customDimension6": "",
+                "customDimension7": "",
+                "customDimension8": "All results: none",
+                "customDimension9": "*",
+                "customDimension10": "",
+                "customDimension11": "",
+                "customDimension12": "",
+                "customDimension13": "",
+                "customDimension14": "",
+                "customDimension15": "",
+                "customDimension16": "",
+                "customDimension17": "",
+                "customMetric1": 10001,
```

Comment:
```suggestion
                "customMetric1": 10000,
```
to resolve and align with issue of consistency for `customMetric1` in this PR.

Reply:
Resolved in [a0ccd87](https://github.com/nationalarchives/ds-wagtail/pull/422/commits/a0ccd87f3aca7f9f87788eaccc56a67f9f38fdb9)